src/redux/actions/notificationActions.js

```diff
@@ -11,3 +13,54 @@ export const fetchNotifications = url => dispatch => {
     });
   };
 };
+
+const dispatchSuccessShowMessage = (dispatch, data) => {
+  dispatch(
+    ShowMessage({
+      message: data.data.data,
```

Comment:
destructuring this would look much cleaner

Reply:
I think is unnecessary since the parameter is only used once.